app/controllers/concerns/application_concern.rb

```diff
@@ -56,6 +61,12 @@ def authenticate_user_in_site
     end
   end
 
+  def track(name, properties = {})
+    ahoy.track name, **properties.merge(site_id: current_site.id)
+  rescue StandardError => e
+    Rollbar.error(e)
```

Comment:
@entantoencuanto Rollbar?

Reply:
I've added the rescue to avoid exceptions trying to generate ahoy events, but the use of Rollbar after the rescue is optional